src/module/module.jl

```diff
@@ -305,6 +305,18 @@ function lift(M::smodule{T}, SM::smodule{T}) where T
    return Module(R, ptr), Module(R,rest_ptr)
 end
 
+@doc Markdown.doc"""
+    lift(M::smodule, SM::smodule, goodShape::Bool, isSB::Bool, divide::Bool)
+
+represents the generators of SM in terms of the generators of M.
+Returns result, rest
+(Matrix(SM) = (Matrix(M)-Matrix(rest))*matrix(result))
```

Comment:
This is exactly the doc string that I was questioning in #452. I think the results satisfy
`Matrix(SM) = Matrix(M)*matrix(result)) + Matrix(rest)`
but what about `U`?